types/yup/yup-tests.ts

```diff
@@ -432,9 +432,56 @@ const description: SchemaDescription = {
     label: 'label',
     meta: { key: 'value' },
     tests: [{ name: 'test1', params: {} }, { name: 'test2', params: {} }],
-    fields: { key: 'value' },
+    fields: { key: { type: 'ref', key: 'value' } },
```

Comment:
Was the previous version invalid with the changes made?

Reply:
`fields` were defined as `object` before, so from TS perspective previous value was correct. But as far as I can see from the code (see attached links above) it is not possible to have a string as a value here. What's more, string makes no sense here.